### PR TITLE
Added config and launch files UR10 and hand lite

### PR DESCRIPTION
**Pull request by beatrizleon**
Gazebo still has problems with this (https://github.com/shadow-robot/sr_interface/issues/136) but it will be ok with real robot

### CODE REVIEW

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_lite_ur10.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+robot:
+  name: ur10srh
```
Comment (kirstyellis) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_lite_ur10.yaml:2]: would it be worth having a different robot name for the ur10 with hand lite?

File: sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<launch>
+  <!-- This launch file is for the right ur10 arm and a shadow hand -->
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch]: add 'lite' here

File: sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  <!-- Set to true to allow planning trajectories-->
+  <arg name="use_moveit" default="true"/>
+  <!--Choose planner library, options are ompl, stomp or sbpl-->
+  <arg name="planning" default="ompl"/>
+  <!-- Set to false if you want to start without guis -->
+  <arg name="gui" default="true"/>
+  <!-- Serial number of the hand to be used-->
+  <arg name="hand_serial" default="1082"/>
+    <!-- the path to the mapping files -->
+  <arg name="mapping_path" default="$(find sr_edc_launch)/mappings/default_mappings/rh_E_v3.yaml"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch]: should the mapping be rh_lite_v1.yaml?

File: sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <arg name="planning" default="$(arg planning)"/>
+    <arg name="gui" default="$(arg gui)"/>
+    <arg name="world" value="$(arg world)"/>
+    <arg name="robot_description" value="$(arg robot_description)"/>
+    <arg name="robot_config_file" value="$(arg robot_config_file)"/>
+    <arg name="hand_id" value="rh"/>
+    <arg name="arm_id" value="ra"/>
+    <arg name="side" value="$(arg side)"/>
+    <arg name="robot_model" value="ur10"/>
+    <arg name="robot_name" value="ur10srh"/>
```
Comment (kirstyellis) [inline sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch]: if you do change the robot name (comment above) it needs to be changed here too

File: sr_multi_description/urdf/right_srhand_lite_ur10_joint_limited.urdf.xacro [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="ur10srh">
```
Comment (kirstyellis) [inline sr_multi_description/urdf/right_srhand_lite_ur10_joint_limited.urdf.xacro:1]: if changing robot name, needs to be changed here too (see next comment)

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_lite_ur10.yaml (hunk 1 above)
Comment (beatrizleon) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_lite_ur10.yaml:2]: It is in different parts of the launch files so I don't think it is not a good idea to change the names.

File: sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch (hunk 3 above)
Comment (beatrizleon) [reply sr_robot_launch/launch/sr_right_ur10arm_lite_hand.launch]: yes, fixed it

**Comment by toliver** [comment]
Why do we need a new launch file? Normally we just pass the desired robot_description to the existing sr_right_ur10arm_hand.launch isn't it?

**Pull request merged**